manifests/init.pp

```diff
@@ -52,6 +52,8 @@
   String                    $user,
   String                    $group,
   Hash[String, Hash]        $instances,
+  String                    $github_domain,
+  String                    $github_api,
```

Comment:
Please, add these two new class parameters in the comments just before the class definition (line 42).

And it would also be interesting to add an example in the README for the GHES scenario here:
https://github.com/Telefonica/puppet-github-actions-runner#hiera-configuration